sdk/keyvault/azure-security-keyvault-certificates/CMakeLists.txt

```diff
@@ -87,7 +87,7 @@ endif()
 # endif()
 
 if(BUILD_SAMPLES)
-   add_subdirectory(samples)
+   add_subdirectory(test/samples)
```

Comment:
Why does it make sense to move these to the `test` directory? Will customers know to be able to find them here?

Reply:
I believe that's the expected repo structure for C++. The samples/ folder next to the src/ and test/ folders provides only docs and link to the sample's src code inside test/ .  I followed .NET here.
The way I see it, we want to use the samples as test-cases, and it is just a test classification ( Functional testing ).

So, I expect to see all types of tests inside the test/ folder ( unit, functional, performance, stress, etc ).   

see .NET:
https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/keyvault/Azure.Security.KeyVault.Keys/samples

Java structure needs to follow the language convention for packages, it places everything within src/
https://github.com/Azure/azure-sdk-for-java/tree/main/sdk/keyvault/azure-security-keyvault-keys/src

Python and JS uses samples/ folder for doscs and code (not in test/ )
https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/keyvault/keyvault-keys/samples/v4


Storage C++ does not follow any known structure ( no surprise hahaha ).  There's a main launcher app for the samples and each sample is just a function.  This makes hard for customers to do copy-paste, but it is easy to maintain for devs :)